public/static/locales/en-US/account.json

```diff
@@ -141,4 +141,4 @@
       }
     }
   }
-}
+}
```

Comment:
Why would you remove the end of file new line?

Reply:
Are you not using Prettier? If you are, then I guess I need to update `.prettierrc`.

Reply:
I am and it is either not loading `.prettierrc.json` or it does not parse `*.json`

Reply:
You can choose “Format document with…” then “Configure default formatter,” if I remember correctly. Make sure it’s Prettier.